.github/workflows/roll-to-exports-branch.yml

```diff
@@ -0,0 +1,59 @@
+# Roll build files to exports branch.
+
+name: Roll to exports branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  roll-to-exports-branch:
+
+    runs-on: ubuntu-latest
+    strategy:
+      # e.g. if lint fails, continue to the unit tests anyway
```

Comment:
remove the fail-fast.

also turns out this only affects matrix jobs and so it's not what we/i commented it as.